### PR TITLE
Add doc for new dapr checkpoint strategy for Azure Event Hubs scalar

### PR DESCRIPTION
**Pull request by brainslush**
Adds documentation for the new dapr checkpoint strategy for Azure Event Hubs scalar. Explains the new option `dapr`  for the `checkpointStrategy` field. Removes dapr reference from `goSdk` checkpoint strategy documentation. 

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda/pull/3139
Relates to https://github.com/kedacore/keda/issues/3141

### CODE REVIEW

File: content/docs/2.8/scalers/azure-event-hub.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ triggers:
 - `checkpointStrategy` - configure the checkpoint behaviour of different Event Hub SDKs. (Values: `azureFunction`, `blobMetadata`, `goSdk`, default: `""`, Optional)
     - `azureFunction` - Suitable for Azure Functions & Azure WebJobs SDK. This is the default setting, when `blobcontainer` is not specified.
     - `blobMetadata` - For all implementations that store checkpoint information on blob metadata such as current C#, Python, Java and JavaScript Event Hub SDKs.
-    - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing, for example Dapr.
+    - `goSdk` - For all implementations using the [Golang SDK](https://github.com/Azure/azure-event-hubs-go)'s checkpointing.
+    - `dapr` - A dapr specific checkpoint strategy based on the `goSdk` checkpointing implementation but using the dapr naming scheme.
```
Comment (tomkerkhove) [inline content/docs/2.8/scalers/azure-event-hub.md:44]: I think the current information is an implementation detail for end-users, what about simply this?
```suggestion
    - `dapr` - Used for Dapr applications using Azure Event Grid.
```

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

**Review by brainslush — COMMENTED** [review]
shortened as requested

**Review by tomkerkhove — APPROVED** [review]
LGTM, thanks!

**Comment by tomkerkhove** [comment]
Done in https://github.com/kedacore/keda-docs/pull/996

**Pull request closed without merging**